services/web-app/components/markdown/image.js

```diff
@@ -24,11 +24,20 @@ const getImageProps = (props) => {
     props.layout = 'responsive'
   }
 
+  // do not optimize if is svg file
+  if (src.split('/').pop().includes('.svg')) {
```

Comment:
```suggestion
  if (src.split('/').pop().endsWith('.svg')) {
```

Reply:
will this work? urls end with `?raw=true`

Reply:
If they end with a query string, then we should parse them via `new URL()` and check the path on that.

Reply:
did that for absolute urls, since these can be relative as well